pkg/scheduler/plugins/reservation/hook.go

```diff
@@ -232,6 +234,27 @@ func preparePreFilterNodeInfo(nodeInfo *framework.NodeInfo, pod *corev1.Pod, mat
 			*podInfo = *newPodInfo
 		}
 	}
+
+	pvcLister := handle.SharedInformerFactory().Core().V1().PersistentVolumeClaims().Lister()
+	for _, rr := range matchedCache.nodeToR[nodeInfo.Node().Name] {
+		podSpecTemplate := rr.Reservation.Spec.Template
+		for _, volume := range podSpecTemplate.Spec.Volumes {
+			if volume.PersistentVolumeClaim == nil {
+				continue
+			}
+			pvc, err := pvcLister.PersistentVolumeClaims(pod.Namespace).Get(volume.PersistentVolumeClaim.ClaimName)
+			if err != nil {
+				continue
+			}
+
+			if !v1helper.ContainsAccessMode(pvc.Spec.AccessModes, v1.ReadWriteOncePod) {
+				continue
+			}
+
+			key := pod.Namespace + "/" + volume.PersistentVolumeClaim.ClaimName
+			nodeInfo.PVCRefCounts[key] -= 1
```

Comment:
What if the nodeInfo does not have a field `PVCRefCount` (e.g. on k8s 1.20)? Could the PVCRefCount be nil?